Icebnb/Icebnb/SearchViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  SearchViewController.swift
+//  Icebnb
+//
+//  Created by Robert Mogos on 05/09/2017.
+//  Copyright © 2017 Robert Mogos. All rights reserved.
+//
+
+import UIKit
+import InstantSearch
+import MapKit
+import Nuke
+import KUIPopOver
+
+class SearchViewController: UIViewController, UICollectionViewDelegate, HitsCollectionViewDataSource {
+  
+  @IBOutlet weak var blurView: UIView!
+  @IBOutlet weak var collectionView: HitsCollectionWidget!
+  @IBOutlet weak var mapView: MapViewWidget!
+  @IBOutlet weak var searchBarView: UISearchBar!
+  var filterBtn: UIButton!
+  var hitsController: HitsController!
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.title = "Icebnb"
+    configureSearchController()
+    configureInstantSearch()
+    configureCollection()
+    configureNavbar()
+    
+    hitsController = HitsController(collection: collectionView)
+    collectionView.dataSource = hitsController
+    collectionView.delegate = hitsController
+    hitsController.collectionDataSource = self
+  }
+  
+  func configureInstantSearch() {
+    InstantSearch.shared.register(searchBar: searchBarView)
```

Comment:
Instead of doing this, you can remove this line, and then in `SearchViewController.xib`, you can add a custom class for the `UISearchBar` to be `SearchBarWidget`. the `registerAll` method will pick it up

Reply:
OK